requirements/dev.txt

```diff
@@ -2,9 +2,12 @@
 -r default.txt
 
 # Extract messages from Django templates
-django-babel==0.4.0 \
---hash=sha256:0cf070be3fc0d8628b008da697aa5c0f89bd62c8dd9238283d5bcdbce70bbec6 \
---hash=sha256:b328dae0e594031d7b0ffded51fd6df60677d4509aee3677013a3a2f76da4e70
+# Code: https://github.com/python-babel/django-babel
+# Docs: http://django-babel.readthedocs.io/en/latest/
+# Changes: https://github.com/python-babel/django-babel/blob/master/CHANGELOG.rst
+django-babel==0.6.2 \
```

Comment:
0.4.0 → [0.6.2](https://github.com/python-babel/django-babel/blob/master/CHANGELOG.rst#062-2017-12-18) trims newlines in translation blocks, and updates support for Python and Django.